### PR TITLE
Emergency attribute update

### PR DESCRIPTION
**Pull request by pkarman**
trello: https://trello.com/c/B8655SyP/253-on-update-cannot-set-request-to-emergency

fixes javascript to actually disable the checkbox on Edit

### CODE REVIEW

**Comment by pkarman** [comment]
codeclimate issue is a red herring

**Comment by yozlet** [comment]
Reviewed and QA'ed, and all looks good, but I suspect this will generate more support load. Further discussion on [the Trello card](https://trello.com/c/B8655SyP/253-on-update-cannot-set-request-to-emergency).

File: app/models/ncr/work_order.rb [hunk 1]
```diff
@@ -31,7 +31,7 @@ def self.purchase_amount_column_name
       with: /\ACL\d{7}\z/,
       message: "must start with 'CL', followed by seven numbers"
     }, allow_blank: true
-    validates :expense_type, inclusion: {in: EXPENSE_TYPES}, presence: true
+    validates :expense_type, inclusion: { in: EXPENSE_TYPES }, presence: true
```
Comment (jessieay) [inline app/models/ncr/work_order.rb]: :heart_eyes:

**Comment by jessieay** [comment]
commented on Trello card. This looks great to me! cc @yozlet

**Comment by yozlet** [comment]
Resolved in the Trello card. Thanks everyone!

**Pull request merged**